recipes/btk/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "btk" %}
+{% set version = "0.4.dev0" %}
+{% set sha256 = "ee067d0e8f0bd0e4087119c71901db7a453b4ea1bc7e616ddb95d808477d1cd5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Biomechanical-ToolKit/BTKCore/archive/v0.3.zip
+  sha256: {{ sha256 }}
+
+# Patches are applied in bld.bat to handle whitespace issues
+#  patches:
+#    - 0001-Updates-since-3.0.patch
+#    - 0002-Python-3-support.patch
+
+
+build:
+  number: 0
+  skip: true  # [not win or py<37]
+
+
+requirements:
+  build:
+    - cmake
+    - jom
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - swig
+    - numpy >=1.14
+    - setuptools
```

Comment:
maybe try removing this dep or replacing it withg pip.  I think you get a slightlu different Python depending on which package you depend on here